package.json

```diff
@@ -11,6 +11,7 @@
     "email": "ericornelissen@gmail.com"
   },
   "scripts": {
+    "preview": "GITHUB_REPOSITORY=simple-icons/simple-icons node src/index.js",
```

Comment:
This environment variable `GITHUB_REPOSITORY`is used?

Reply:
Yes, it's used by the octokit. You can try without this variable to see the error.